include/cetl/unbounded_variant.hpp

```diff
@@ -520,10 +1091,9 @@ class unbounded_variant : detail::base_move<Footprint, Copyable, Movable, Alignm
     /// \param args Arguments to be forwarded to the constructor of `ValueType`.
     ///
     template <typename ValueType, typename... Args, typename Tp = std::decay_t<ValueType>>
-    Tp& emplace(Args&&... args)
+    Tp* emplace(Args&&... args)
```

Comment:
Now we support possible PMR failures, so in case c++ are off you get `nullptr` result from `emplace`).